src/append_only_data.rs

```diff
@@ -581,7 +596,7 @@ macro_rules! impl_appendable_data {
                 self.inner.permissions.get(index)
             }
 
-            /// Returns the owner's public key and the indices at the time it was added.
+            /// Returns the owner's public key and the indices at the time it was added. // <--- confusing docs
```

Comment:
I'm not sure if you meant to commit this, but won't this comment appear in the public documentation for this function? I would say either move it into a separate `TODO:` comment or just address it now.

Reply:
I did not actually, I'm sorry it slipped through. It was a note to myself to get to the bottom with what it meant.